tests/helpers/drupal-commands.js

```diff
@@ -5,15 +5,14 @@ function drush(command) {
 module.exports = {
   /**
    * Finds node ID via drush and visits node edit page.
-   * @param  {Array<{page: Page, node_title: String}>} array Page object and
-   *   node title
-   * @param {string} langcode Optional language code prefix for the URL.
+   * @param  {Array<{page: Page, node_title: String, langcode: String}>} array
+   *   Page object, node title and optional language code prefix for the URL.
    *   Note the node title must be in the original language. The visited page
    *   will edit the node's translation if it exists, otherwise the original
    *   language node (just in another UI language).
    * @return {Response} The response.
    */
-  visitNodeEditPage: async ([page, node_title], langcode = '') => {
+  visitNodeEditPage: async ([page, node_title, langcode = '']) => {
```

Comment:
Testing @sindurigf 's change made me realize that I don't know Javascript _(well I knew that part)_ and that it is fine to have an optional part of an array, _inside_ an array parameter.

In other words, this change is possible and does not change anything - and that means the recent addition that I did of langcode _outside of the array_ is unnecessary... and inconsistent.

So I 'fixed' that in this same PR.

Reply:
None of the current `visitNodeEditPage()` calls pass the optional `langcode`, so nothing breaks. (This was added for LDP-1787 which isn't merged yet.)